colcon.pkg

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "xtypes",
+    "type": "cmake",
+    "dependencies" : ["cpp-peglib"]
```

Comment:
I do not agree. Taking the `.h` file is correct when you know the exact path of the file (thirdparty case) but not when it is an external library. In that case, the library must be compiled.
If the library is a header only, it would not take time. If it is not, we may think another option to get this `.h` (In FastDDS, is insert in thirdparty as itself; other option would be a vendor).